### PR TITLE
Correcting some Namespaces

### PR DESCRIPTION
**Pull request by andreiborodin**
This work is two fold:
- Correct some non-updated namespaces
- Put some classes into a namespace (this otherwise would be errors in our NuGet retargeting step)

### CODE REVIEW

File: Assets/MixedRealityToolkit/Interfaces/Devices/IMixedRealityHandVisualizer.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Input;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.Devices
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Interfaces/Devices/IMixedRealityHandVisualizer.cs]: None of these are specific to WindowsMixedReality, so they should probably be moved into just `Toolkit.Input`, with the rest of the interfaces and profiles.

File: Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityEyeTrackingProfileInspector.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.﻿
 
-using Microsoft.MixedReality.Toolkit.Core.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input;
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityEyeTrackingProfileInspector.cs]: This one

File: Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityEyeTrackingProfile.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Utilities;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Definitions.Devices
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityEyeTrackingProfile.cs]: This one

File: Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityEyeSaccadeProvider.cs [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityEyeSaccadeProvider.cs]: This one

File: Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/AttachToControllerInspector.cs [hunk 5]
```diff
@@ -6,7 +6,7 @@
 using UnityEditor;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.UI.Inspectors.Utilities.Solvers
+namespace Microsoft.MixedReality.Toolkit.UI.Editor.Solvers
```
Comment (wiwei) [inline Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/AttachToControllerInspector.cs]: I know that this was already discussed on the chat, but I do wanna call out that some other solvers are in this namespace:

namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers

[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs)
[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs)
[https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Overlap.cs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Overlap.cs)

I don't really have a strong opinion between any of these so the power is yours.

Comment (andreiborodin) [reply Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/AttachToControllerInspector.cs]: I will revisit all solvers namespace tomorrow.

---
In reply to: [276877813](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/4019#discussion_r276877813) [](ancestors = 276877813)

Comment (wiwei) [reply Assets/MixedRealityToolkit.SDK/Inspectors/Utilities/Solvers/AttachToControllerInspector.cs]: Also, facepalm on my part, I totally missed the inspector word =/

---
In reply to: [276878366](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/4019#discussion_r276878366) [](ancestors = 276878366,276877813)

**Pull request merged**